apps/crn-server/src/handlers/event-bus.ts

```diff
@@ -42,7 +42,15 @@ export type ExternalAuthorPayload =
   | ExternalAuthorSquidexPayload
   | ExternalAuthorContentfulPayload;
 
-export type GroupPayload = SquidexWebhookPayload<Group, GroupEvent>;
+export type GroupSquidexPayload = WebhookDetail<
+  SquidexWebhookPayload<Group, GroupEvent>
+>;
+
+export type GroupContentfulPayload = WebhookDetail<
+  ContentfulWebhookPayload<'interestGroups'>
+>;
+
+export type GroupPayload = GroupSquidexPayload | GroupContentfulPayload;
```

Comment:
Maybe we could rename it to `InterestGroupPayload` and the other ones as well? :)